src/delivery_reservation_lamp_manager.cpp

```diff
@@ -38,13 +43,26 @@ DeliveryReservationLampManager::DeliveryReservationLampManager(
   // Timer
   std::chrono::milliseconds timer_period_msec;
   timer_period_msec = std::chrono::duration_cast<std::chrono::milliseconds>(
+    std::chrono::duration<double>(1.0 / 10.0));
+  
+  auto timer_callback = std::bind(&DeliveryReservationLampManager::onTimer, this);
```

Comment:
We want the naming to make sense when viewed side-by-side with blink_timer_callback.
How about naming it "main_proc_timer_callback" since it will be the main process?

Reply:
Fixed name.